qml/js/youtube/YouTubeCipher.js

```diff
@@ -86,6 +86,6 @@ function decodeCipheredVideo(videoid, mediagrabber, urldecoder)
         }
     }
 
-    req.open("GET", "http://www.youtube.com/watch?v=" + videoid + "&gl=US&persist_gl=1&hl=en&persist_hl=1");
+    req.open("GET", "https://www.youtube.com/watch?v=" + videoid + "&gl=US&persist_gl=1&hl=en&persist_hl=1");
```

Comment:
Did you ever test this or was there something else wrong?

Typing it to DuckDuckGo offered Wikipedia page https://en.wikipedia.org/wiki/XMLHttpRequest which again says:

> XMLHttpRequest (XHR) is an API available to web browser scripting languages such as JavaScript. It is used to send **HTTP or HTTPS** requests to a web server and load the server response data back into the script.